packages/serverless/src/common/types.ts

```diff
@@ -14,6 +14,12 @@ export type TAuthorizedContext = TPublicContext & {
   email: string;
 };
 
+export type PermissionBasic = {
```

Comment:
Change to `PermissionAndOwnership` in `@zkdb/permission`

Reply:
It not satisfied the type that I provide